components/cli/cmd/cellery/setup_cleanup_gcp.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package main
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/constants"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/commands"
+)
+
+func newSetupCleanupGcpCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "gcp",
+		Short: "Cleanup gcp setup",
+		Args: func(cmd *cobra.Command, args []string) error {
+			err := cobra.ExactArgs(1)(cmd, args)
+			if err != nil {
+				return err
+			}
+			return commands.ValidateGcpCluster(args[0])
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			uniqueNumber := strings.Split(args[0], constants.GCP_CLUSTER_NAME)[1]
```

Comment:
Shall we check the slice length return from the Split function? Otherwise using index 1 directly may cause "slice bounds out of range" error

Reply:
updated